<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAccount</class>
 <widget class="QWidget" name="PageAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF</string>
  </property>
  <widget class="QLabel" name="historyLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>195</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: transparent;
	color: #474D57;
	}</string>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="receivePushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FCD535;
	color: #474D57;
	border-style: solid;
	border-width: 4px;
	border-radius: 29px;
	border-color: #F4F5F5;
}
QPushButton::hover {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFEF4F, stop: 0.4 #FFEF4F, stop: 0.5 #FFEF4F, stop: 1.0 #FFEF4F);
}
QPushButton::pressed {
	border-color: #FCD535;
	background-color: #FCD535;
	border-style: solid;
	border-width: 4px;
	border-color: #F4F5F5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/ic/res/ic/arrowDown.png</normaloff>:/res/ic/res/ic/arrowDown.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="accountValueLyrLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: transparent;
	color: #474D57;
}</string>
   </property>
   <property name="text">
    <string>0.00000000 LYR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="historyPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/ic/res/ic/article.ico</normaloff>:/res/ic/res/ic/article.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lyraLogoPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: transparent;
	border-style: solid;
	color: #474D57;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/ic/res/ic/ic_lyra_token_foreground.png</normaloff>:/res/ic/res/ic/ic_lyra_token_foreground.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="accountTotalValueUsdLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: transparent;
	color: #474D57;
}</string>
   </property>
   <property name="text">
    <string>Total balance: $ 0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="accountValueUsdLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: transparent;
	color: #474D57;
}</string>
   </property>
   <property name="text">
    <string>0.00 USD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FCD535;
	color: #474D57;
	border-style: solid;
	border-width: 4px;
	border-radius: 29px;
	border-color: #F4F5F5;
}
QPushButton::hover {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFEF4F, stop: 0.4 #FFEF4F, stop: 0.5 #FFEF4F, stop: 1.0 #FFEF4F);
}
QPushButton::pressed {
	border-color: #FCD535;
	background-color: #FCD535;
	border-style: solid;
	border-width: 4px;
	border-color: #F4F5F5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/ic/res/ic/arrowUp.png</normaloff>:/res/ic/res/ic/arrowUp.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
